nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -678,15 +678,14 @@ private Description checkOverriding(
       Symbol.MethodSymbol overridingMethod,
       @Nullable MemberReferenceTree memberReferenceTree,
       VisitorState state) {
-    // We ignore unannotated methods for now, since we always assume they return @NonNull, but we
-    // don't actually know,
-    // so it might make sense to override them as @Nullable.
-    // TODO: Add an equivalent to Handler.onUnannotatedInvocationGetExplicitlyNullablePositions for
-    // @NonNull return
-    // values if needed.
+    final boolean isOverridenMethodUnannotated =
+        NullabilityUtil.isUnannotated(overriddenMethod, config);
     boolean overriddenMethodReturnsNonNull =
```

Comment:
Actually, I like your `final` above, can we make this `final` too?

Reply:
done